app/controllers/staffController.js

```diff
@@ -264,5 +264,39 @@ export default class StaffController {
         message: 'you must be a staff (Admin) to perform this task',
       });
     }
+
+    // Specific Bank Account
+
+    static async getUserAccounts(req, res) {
+      const user = auth.tokenBearer(req);
+      if (user.isAdmin && user.type.toLowerCase() === 'staff') {
+        const allAccountsQuery = 'SELECT * FROM accounts WHERE owner = (SELECT user_id FROM users WHERE email = $1)';
```

Comment:
You can create a file called query where all your queries will reside, instead of writing raw queries in controller

Reply:
Ok, I will implement it thanks